app/models/user.rb

```diff
@@ -113,7 +114,11 @@ def confirmable_template
   end
 
   def was_migrated_from_softwire?
-    old_user_id.present?
+    legacy_users.any?
+  end
+
+  def old_user_id
```

Comment:
I think this method should probably just be removed altogether since it's original purpose isn't really valid anymore any remaining usages are likely a mistake.

Reply:
That's a great point. I've removed this method and updated `was_migrated_from_softwire?` above.

Reply:
Fixed in [e54b9d0](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/894/commits/e54b9d0d59e2a080e82aa7dcddf0a1b5171ae0c3)